docs/docs/introduction.mdx

```diff
@@ -6,11 +6,22 @@ description: Learn more about open-source natural language processing library Ra
 abstract: Rasa is an open source machine learning framework for automated text and voice-based conversations. Understand messages, hold conversations, and connect to messaging channels and APIs.
 ---
 
-Let's get started and [prototype an assistant](prototype-an-assistant.mdx)!
+Rasa Open Source includes:
+- NLU: determines what the user wants and captures key contextual information
+- Dialogue management: selects the next best response or action based on conversation history
+- Channels and integrations: connect to users and backend systems
+
+## Prototype an Assistant
+
+Try out Rasa Open Source before you install by [prototyping an assistant](prototype-an-assistant.mdx)!
```

Comment:
Added some text here because we don't have a nice external link component -- but it might be nice anyways to give some context of what prototyping an assistant is or why its a good next step

Reply:
Yep, this looks good! We should still add some styling for an external link component (will try come up with something), but this is a good first step